lib/fedex/request/base.rb

```diff
@@ -219,7 +228,7 @@ def add_packages(xml)
       
       def add_freight_shipment_detail(xml)
         xml.FreightShipmentDetail {
-          if shipping_from_our_warehouse?
+          # if shipping_from_our_warehouse?
```

Comment:
Is this related?

Reply:
oh yeah, i need to just remove this all completely. basically, we're always going to be billing collect when it's inbound, and collect is incompatible with an alternate billing method.

Reply:
(also all this stuff should go into a wrapper for the main repo instead of here)